database/queries.sql

```diff
@@ -0,0 +1,18 @@
+select items.item_name
```

Comment:
it's a common practice to use upper case for reserved keywords, so this should be:
```
SELECT items.item_name
FROM items JOIN wishlist USING (itemid)
WHERE userid = 1;
```